### DISCUSSION TITLE
Cable length?

### DISCUSSION THREAD

**Discussion by reedy** [revk/xESP32-Faikout#340] [Q&A]
Can anyone advise roughly what length of cable they've needed between the S21 port and the Faikin?

I've not seemingly found it documented, but could be useful information especially if buying pre-terminated cables, for example.

Thanks!

**Comment by benswinburne** [comment]
I have a 6” cable on my FTXS09LVJU. It comfortable reaches from the s21 port at the bottom of the main board through a small hole into the rear bottom of the external plastic case. It reaches away from the metal housing but far enough into the plastic housing. 

You could probably go shorter if you were comfortable with the main board and the faikin staying together inside the metal area with the main board but obviously that’d be detrimental to the wireless signal.

Longer cables you can just tie up the excess in a loop.
